packages/agreement-process/test/upgradeAgreement.test.ts

```diff
@@ -554,335 +545,6 @@ describe("upgrade Agreement", () => {
     }
   });
 
-  it("should succeed with valid Verified, Certified, and Declared attributes when consumer and producer are different and requester is a delegate", async () => {
```

Comment:
See comment in https://github.com/pagopa/interop-be-monorepo/pull/1260, this test was not needed and is being removed also on `develop`.